make/deps.mk

```diff
@@ -27,6 +27,7 @@ DEPS = amqp_client \
 	jiffy \
 	lager \
 	lager_syslog \
+	meta \
```

Comment:
Where is this being used? I didn't see any calls to `forms` or `meta` (but easily could have missed them)

Reply:
is a secret